docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -103,19 +103,21 @@ but they are created with `NoExecute` tolerations for the following taints with
 
  - `node.alpha.kubernetes.io/notReady`
  - `node.alpha.kubernetes.io/unreachable`
- - `node.alpha.kubernetes.io/memoryPressure`
- - `node.alpha.kubernetes.io/diskPressure`
-
-When the support to critical pods is enabled and the pods in a DaemonSet are
-labelled as critical, the Daemon pods are created with an additional
-`NoExecute` toleration for the `node.alpha.kubernetes.io/outOfDisk` taint with
-no `tolerationSeconds`.
 
 This ensures that when the `TaintBasedEvictions` alpha feature is enabled,
 they will not be evicted when there are node problems such as a network partition. (When the
 `TaintBasedEvictions` feature is not enabled, they are also not evicted in these scenarios, but
 due to hard-coded behavior of the NodeController rather than due to tolerations).
 
+ They also tolerate following `NoSchedule` taints:
+ - `node.kubernetes.io/memoryPressure`
+ - `node.kubernetes.io/diskPressure`
```

Comment:
https://github.com/kubernetes/kubernetes/issues/51246 proposed a new name of those taints, should we update them in 1.8.

Reply:
maybe we can just fix line 106-107 instead of introducing a new list for user to merge in mind?

Reply:
Those are completely different things and separate features which can be enabled separately. I think the danger of mixing them is bigger than inconvenience of having two separate lists.

Reply:
oh really?
The code seems make no differentiation among them:
https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/daemon/daemon_controller.go#L1147